db/schema.rb

```diff
@@ -33,7 +33,7 @@
     t.index ["expires_on"], name: "index_approvals_on_expires_on"
   end
 
-  create_table "attendances", id: :uuid, default: -> { "public.gen_random_uuid()" }, force: :cascade do |t|
```

Comment:
@twinsfan421 can you double check your postgres version and try and run `bundle exec rails db:migrate` again?  This schema prefix removal might be a side effect of a different postgres version.  I'm running Postgres 13 using Postgres.app on my MacBook Pro.  I did about 5 mins of research and couldn't find a quick answer to which version or configuration retains the prefixes. If you can't find a way to prevent it, could you remove these changes from your commit?

Reply:
@katelovescode  I just pushed up the fixes.  Thanks